src/odemis/acq/fastem.py

```diff
@@ -401,13 +419,19 @@ def run(self):
             raise
 
         except Exception as ex:
-            # Check if any field images have already been acquired; if not => just raise the exception.
-            if len(self._fields_remaining) == len(self._roa.field_indices):
-                raise
-            # If image data was already acquired, just log a warning.
-            logging.warning("Exception during roa acquisition (after some data has already been acquired).",
-                            exc_info=True)
-            exception = ex  # let the caller handle the exception
+            if self._stop_acq_on_failure:
+                # Check if any field images have already been acquired; if not => just raise the exception.
+                if len(self._fields_remaining) == len(self._roa.field_indices):
+                    raise
+                # If image data was already acquired, just log a warning.
+                logging.warning("Exception during roa acquisition (after some data has already been acquired).",
+                                exc_info=True)
+                exception = ex  # let the caller handle the exception
+            else:
+                exception = ROASkipped(
+                    f"Skipped the ROA {self._roa.shape.name.value} due to acquisition failure with exception: {ex}"
+                )
+                logging.warning("%s", exception)
```

Comment:
This drops the whole backtrace. Although the acquisition continues, it might be quite valuable to know the context of the failure.  => add a `exc_info=True` here too.